src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    public static void main(String[] args) {
+        Random random = new Random();
+        ColorSupplier cs = new ColorSupplier(random, Colors.values());
+        FigureSupplier fs = new FigureSupplier(random, cs, Figures.values());

```

Comment:
We can access enums from any part of the project directly 
```suggestion
        ColorSupplier cs = new ColorSupplier(random);
        FigureSupplier fs = new FigureSupplier(random, cs);
```